### ISSUE TITLE
mouseover text

### ISSUE THREAD

**Issue by Pockets-byte** [unitystation/unitystation#6071]
Mouseover text is a holdover from byond, which is a holdover from ancient windows systems. and is not suited for a fast-paced video game. it should be parented to the mouse. and next to it. so the text is always where the player's focus is.

**Comment by ThatDan123** [comment]
It should not be instantaneous, maybe hold over same position for 2 seconds before tooltip appears (configurable in settings, also keep the old version available too as an option)

**Comment by corp-0** [comment]
I'd like to experiment with it before discarding the instantaneous part. Maybe it isn't that bad with something like half a second.

**Comment by ThatDan123** [comment]
yeah, at least some sort of delay, as moving the mouse around the screen and it popping up everything would be annoying imo

**Comment by Pockets-byte** [comment]
I think it defeats the purpose to have it on a timer. And in my personal opinion a single line of text next to the mouse would be un-intrusive enough. However I know that not everyone prefers that solution. So my suggestion is to have it configurable. Delay and position, and after like a week of having it in, we hold a poll to determine the default setting 

we should also consider by hotkey as well, both held and toggle, as was previously suggested

**Comment by CdrCross** [comment]
Take a cue from recent Paradox games, the masters of tooltips (as this mouseover text is commonly known): Let the user configure the delay. The standard in Paradox's games is 0.5sec, but let the user tweak it.

**Comment by Pockets-byte** [comment]
I honestly don't consider tooltips and the mouseover text to be the same
thing. consider Waila vs tooltips in minecraft, one is always there and
instant. while the other only applies to items in your inventory
